pkg/components/prometheus-operator/component.go

```diff
@@ -55,13 +55,17 @@ type component struct {
 	PrometheusMetricsRetention  string            `hcl:"prometheus_metrics_retention,optional"`
 	PrometheusExternalURL       string            `hcl:"prometheus_external_url,optional"`
 	PrometheusNodeSelector      map[string]string `hcl:"prometheus_node_selector,optional"`
+	PrometheusStorageSize       string            `hcl:"prometheus_storage_size,optional"`
```

Comment:
Why string? it should be int. Or the default value should be `50 Gi`.

Reply:
I don't think I understand what you mean?

It is string in the end, isn't it?

```console
$ kubectl explain pvc.spec.resources.requests.storage
KIND:     PersistentVolumeClaim
VERSION:  v1

FIELD:    storage <string>

DESCRIPTION:
     Quantity is a fixed-point representation of a number. It provides
...
```

Reply:
I mean, if we ask user for size in GiB, we should take it as `int`, as string in this case is a weaker type. If we allow user to specify the unit, it can stay as string then, but we should fix the default value and the documentation.

Reply:
changed, PTAL